api/docs/vscode-azureresources-api.d.ts

```diff
@@ -48,7 +48,7 @@ export declare enum AzExtResourceType {
     StaticWebApps = "StaticWebApps",
     StorageAccounts = "StorageAccounts",
     VirtualMachines = "VirtualMachines",
-    ResourceGroup = "ResourceGroup",
+    ResourceGroup = "ResourceGroups",

```

Comment:
Azure returns a resource group as `resourceGroups` in the Azure ID, so it makes sense to add the s at the end here.